src/client.ts

```diff
@@ -63,6 +63,7 @@ export class SkynetClient {
 
   // SkyDB
   db = {
+    keyPairFromSeed: keyPairFromSeed.bind(this),
```

Comment:
I would not define this on the db.

Reply:
agreed, it's also something registry will make use of